ivy/container/general.py

```diff
@@ -2529,6 +2558,31 @@ def value_is_nan(
         ret
             Boolean as to whether the input value is a nan or not.
 
+        Examples
+        --------
+        >>> x = ivy.Container(a=ivy.array([425]), b=ivy.array([float('nan')]))
+        >>> y = x.value_is_nan()
+        >>> print(y)
+        {
+            a: False,
```

Comment:
somehow the generated strings for bool representation is lowercase (i.e., `false`, `true`), which currently causes doctoring test failures, would you mind changing the container representation returns to be lowercase? only the examples here, thanks a lot!